src/map/elemental.hpp

```diff
@@ -42,22 +43,19 @@ enum elemental_elementalid  : uint16 {
 };
 
 struct elemental_skill {
```

Comment:
```suggestion
struct s_elemental_skill {
```
This will require a few other changes as well if it wasn't visible in the PR already. I tried marking all spots I could see here.